homeassistant/components/home_connect/const.py

```diff
@@ -52,3 +52,5 @@
 ATTR_SIGN = "sign"
 ATTR_UNIT = "unit"
 ATTR_VALUE = "value"
+
+OPERATION_STATE_DEVICE_CLASS = "home_connect__operation_state"
```

Comment:
We don't use custom device classes for translation in Core integrations anymore.

See:  <https://developers.home-assistant.io/docs/internationalization/core#state-of-entities>